src/components/TileGallery/_tile-gallery.scss

```diff
@@ -152,7 +152,7 @@
       justify-content: center;
       align-items: center;
       background: $ui-02;
-      margin-bottom: 0.9375rem;
+      margin-bottom: $spacing-05;
```

Comment:
Does not map to original value. Seems awfully specific rem value which may mean a break in layout

Reply:
Making an exception for this one 👍